src/components/Schedule/ReservationInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from "react";
+
+interface IProps {
+  date: string;
+  therapyName: string;
+  doctorName: string;
+  place: string;
+  therapyLength: number;
+  price: number;
+}
+
+export const ReservationInfo: FC<IProps> = ({
+  date,
+  therapyName,
+  doctorName,
+  place,
+  therapyLength,
+  price
+}) => (
+  <div className="pt-16">
+    <div className="flex">
+      <div className="text-21 font-gilbert border-solid pt-px pr-8 pb-8 border-lilac border-r">
+        {date}
+      </div>
+      <div className="text-21 ml-8 mt-px font-gilbert">{therapyName}</div>
+    </div>
+    <div className="flex flex-row items-center">
+      <div
+        style={{ marginLeft: "54px" }}
+        className="text-12 font-lato text-grey border-solid p-4 pr-8 border-lilac border-r"
```

Comment:
you can apply font-lato in the root div of component and this font will be applied in entire component (except of the elements, where you define another font class). So, you can apply the font that you use most often just once in the top of the component and then just apply another font where you need it (please check this repeating of fonts in entire pr)